src/scripting_api/aform.js

```diff
@@ -651,7 +675,7 @@ class AForm {
       case 2:
         const value = this.AFMergeChange(event);
         formatStr =
-          value.length > 8 || value.startsWith("(")
+          value.startsWith("(") || (/^\p{N}+$/.test(value) && value.length > 7)
```

Comment:
Maybe swap the second part of this condition around, since the length-check is probably faster than the regular expression?
```suggestion
          value.startsWith("(") || (value.length > 7 && /^\p{N}+$/.test(value))
```